README.md

```diff
@@ -27,6 +27,68 @@ This chart bootstraps a [Mender](https://mender.io) deployment on a [Kubernetes]
 - Kubernetes 1.12+
 - Helm 2.11+ or Helm 3.0-beta3+
 
+## External services required
+
+This Helm chart does not install the following external services and dependencies which are required to run Mender:
+
+- mongodb
+- MinIO
+- NATS
+
+### Installing mongodb
+
+You can install mongodb using the official mongodb helm chart using `helm3`:
+
+```bash
+$ helm repo add bitnami https://charts.bitnami.com/bitnami
+$ helm repo update
+$ helm install mongodb --set "auth.enabled=false" bitnami/mongodb
+```
+
+or using `helm2`:
+
+```bash
```

Comment:
I'd consider if we still want to support helm2 anyhow. Its not supported in the upstream anymore and charts might be behaving weird if they start to rely on helm3 features